<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frm_Filter</class>
 <widget class="QWidget" name="Frm_Filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>43</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SingleFilter</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QComboBox" name="cboxField"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="stckFilterPage">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_9">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>  is  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cboxChoices"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>229</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="cboxStringOper">
         <item>
          <property name="text">
           <string>contains</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>begins with</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ends with</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>is</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>is not</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edString"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>109</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_5">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="cboxNumberOper">
         <item>
          <property name="text">
           <string>equals</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>less than</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>greater than</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>is not</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edNumber"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>89</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_6">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="cboxDateCriteria">
         <item>
          <property name="text">
           <string>within last</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>exactly</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>before</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>after</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>today</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>yesterday</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>this week</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>this month</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>this year</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stckDateTimeFilterValue">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QSpinBox" name="spboxTimeUnits"/>
           </item>
           <item>
            <widget class="QComboBox" name="cboxTimeUnits">
             <item>
              <property name="text">
               <string>hours</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>days</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>weeks</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>months</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>years</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_3">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QDateTimeEdit" name="dtDateTime"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_7"/>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="tbtnAdd">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>➕</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tbtnRemove">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>➖</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
